packages/flutter_tools/test/integration.shard/flutter_immediately_exit_test.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:vm_service/vm_service.dart';
+import 'package:vm_service/vm_service_io.dart';
+
+import '../src/common.dart';
+import 'test_data/project_with_immediate_exit.dart';
+import 'test_driver.dart';
+import 'test_utils.dart';
+
+void main() {
+  Directory tempDir;
+  final ProjectWithImmediateExit _project = ProjectWithImmediateExit();
+  FlutterRunTestDriver _flutter;
+
+  setUp(() async {
+    tempDir = createResolvedTempDirectorySync('run_test.');
+    await _project.setUpIn(tempDir);
+    _flutter = FlutterRunTestDriver(tempDir);
+  });
+
+  tearDown(() async {
+    tryToDelete(tempDir);
+  });
+
+
+  testWithoutContext('flutter_tools gracefully handles quick app shutdown', () async {
```

Comment:
Nice! Is it possible to use the test driver with this code? I believe it will throw an exception if the startup fails, but you could still catch that and inspect the logs

Reply:
Took a couple changes, but got that working. Didn't see it the first time around.